exporter/jaeger/jaegerthrifthttpexporter/factory_test.go

```diff
@@ -98,7 +98,7 @@ func TestFactory_CreateTraceExporterFails(t *testing.T) {
 					NameVal: typeStr,
 				},
 			},
-			errorMessage: "\"jaeger_thrift_http\" config requires a valid \"url\": parse : empty url",
+			errorMessage: "\"jaeger_thrift_http\" config requires a valid \"url\": parse \"\": empty url",
```

Comment:
nit: since touching these could use back-tick to avoid escaping the double-quotes

Reply:
Not sure I understand, but happy to review a PR :)